CASC/PortfolioMode.cpp

```diff
@@ -378,7 +380,34 @@ bool PortfolioMode::runSchedule(Schedule& schedule)
   PortfolioSliceExecutor executor(this);
   ScheduleExecutor sched(&policy, &executor);
 
-  return sched.run(schedule);
+  bool result = sched.run(schedule);
+
+  //All children have been killed. Now safe to print proof
+  if(result && !env.options->printProofToFile()){
+    vstring fname = env.options->problemName() + "-vampire.proof";    
```

Comment:
Just an idea: this looks like the proof is to be written to a place where the benchmark lies. This could often be a read only location...

Maybe we should force the user to specify the output location whenever `printProofToFile` is true (making the option a string directly). And maybe we should use the temp folder or at least the current vampire's directory (and a name which contains the process's pid) for when it it false (and we still need a file, like this).

Reply:
Update: the user is now given the option to specify the output location. If they leave it blank, the output location is the directory from which the run command happens. Likewise, this is the directory used when `printProofToFile` is false.